tests/integration/test_messages_integration.py

```diff
@@ -1306,6 +1306,34 @@ def test_capture_delays(tmp_path, loops, sleep, max_repetitions):
         pytest.fail(f"Delayed capture: {too_big} avg delay is {sum(delays) / len(delays):.3f}")
 
 
+@pytest.mark.parametrize("output_is_terminal", [True])
+def test_progress_and_message(capsys, logger):
+    emit = Emitter()
+    emit.init(EmitterMode.BRIEF, "testapp", GREETING)
+    emit.progress("The meaning of life is 42.")
+    emit.progress("Another message.")
+    emit.message("Finished successfully.")
+    emit.ended_ok()
+
+    expected_term = [
+        Line("The meaning of life is 42.", permanent=False),
+        Line("Another message.", permanent=False),
+    ]
+    expected_out = [Line("Finished successfully.", permanent=True)]
```

Comment:
@sergiusens without the fix, this line would've been `"Another message. :: Finished successfully."`